interpolated_model_timeseries.py

```diff
@@ -1,96 +1,81 @@
 #!/usr/bin/env python3
-# -*- coding: utf-8 -*-
-"""
-Created on Tue Jan 21 13:32:52 2020
-
-Concatenates a list of wrfout files, then interpolates timeseries at given 
-lat, lon locations
-The wrfout files must have been regridded to a rectilinear grid
-Returns a dataframe with the interpolated timeseries
-
-@author: eebjs
-"""
-
 import xarray as xr
 import pandas as pd
 import numpy as np
 from scipy.interpolate import RegularGridInterpolator
 
-### Inputs ####################################################################
-
-# a list of lat, lon tuples
 locations = [(25, 100), (26, 101), (27, 102), (28, 103), (29, 101), (30, 101)]
-
-# a list of filepaths to regridded wrfout files
 wrfouts = ['/nfs/a68/eebjs/wrfoutput/p2run/202/ctl2016/feb/wrfout_feb2016_ctl_rgdd25.nc',
            '/nfs/a68/eebjs/wrfoutput/p2run/202/ctl2016/mar/wrfout_mar2016_ctl_rgdd25.nc',
            '/nfs/a68/eebjs/wrfoutput/p2run/202/ctl2016/apr/wrfout_apr2016_ctl_rgdd25.nc',
            '/nfs/a68/eebjs/wrfoutput/p2run/202/ctl2016/may/wrfout_may2016_ctl_rgdd25.nc']
-
 #wrfouts = ['/nfs/a336/earlacoa/paper_aia_china/test_file/wrfout_combined-domains_global_0.25deg_2015-01_PM2_5_DRY.nc']
-
-# a string of the data variable that should be extracted from the wrfout file 
 pol = 'PM2_5_DRY'
-
-# an integer representing the model level (z dim) to interpolate at 
 level = 0
 
-# name of co-ordinate used for time dimension
-time = 'Time'
+with xr.open_dataset(wrfouts[0]) as ds:
+    if 'time' in ds[pol].coords:
+        tdim = 'time'
```

Comment:
This is a good way of doing it. From now on when I concatenate wrfout files I will stick to your file conventions so hopefully we can remove the need for this bit of cod